src/dev/impl/DevToys/ViewModels/Tools/Generators/PasswordGenerator/PasswordGeneratorToolProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Composition;
+using DevToys.Api.Tools;
+using DevToys.Shared.Api.Core;
+
+namespace DevToys.ViewModels.Tools.Generators.PasswordGenerator
+{
+    [Export(typeof(IToolProvider))]
+    [Name("Password Generator")]
+    [Parent(GeneratorsGroupToolProvider.InternalName)]
+    [ProtocolName("password")]
+    [Order(1)]
+    [CompactOverlaySize(width: 400, height: 500)]
+    internal class PasswordGeneratorToolProvider : IToolProvider
+    {
+        private readonly IMefProvider _mefProvider;
+
+        public string MenuDisplayName => LanguageManager.Instance.PasswordGenerator.MenuDisplayName;
+
+        public string? SearchDisplayName => LanguageManager.Instance.PasswordGenerator.SearchDisplayName;
```

Comment:
This string (and the other below) shouldn't be nullable.

Reply:
fixed